docs/src/components/productX/XHero.tsx

```diff
@@ -0,0 +1,193 @@
+import * as React from 'react';
+import { DataGridPro } from '@mui/x-data-grid-pro';
+import { useDemoData } from '@mui/x-data-grid-generator';
+import Box from '@mui/material/Box';
+import Divider from '@mui/material/Divider';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
+import { DateRange } from '@mui/lab/DateRangePicker';
+import StaticDateRangePicker from '@mui/lab/StaticDateRangePicker';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import GradientText from 'docs/src/components/typography/GradientText';
+import GetStartedButtons from 'docs/src/components/home/GetStartedButtons';
+import HeroContainer from 'docs/src/layouts/HeroContainer';
+import IconImage from 'docs/src/components/icon/IconImage';
+import FolderTreeView from 'docs/src/components/showcase/FolderTreeView';
+
+const startDate = new Date();
+startDate.setDate(10);
+const endDate = new Date();
+endDate.setDate(endDate.getDate() + 28);
+
+export default function XHero() {
+  const { loading, data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 10000,
+    maxColumns: 40,
+    editable: true,
+  });
+  const [value, setValue] = React.useState<DateRange<Date>>([startDate, endDate]);
+  return (
+    <HeroContainer
+      left={
+        <Box sx={{ textAlign: { xs: 'center', md: 'left' } }}>
+          <Typography
+            fontWeight="bold"
+            variant="body2"
+            color={(theme) => (theme.palette.mode === 'dark' ? 'primary.400' : 'primary.600')}
+            sx={{
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: { xs: 'center', md: 'flex-start' },
+              '& > *': { mr: 1, width: 28, height: 28 },
+            }}
+          >
+            <IconImage name="product-advanced" /> MUI X
+          </Typography>
+          <Typography variant="h1" sx={{ my: 2, maxWidth: 500 }}>
+            Performant
+            <br />
+            <GradientText>advanced</GradientText>
+            <br /> components
+          </Typography>
+          <Typography color="text.secondary" sx={{ mb: 3, maxWidth: 500 }}>
+            Build complex and data-rich applications using a growing list of advanced components.
+            We&apos;re kicking it off with the most powerful Data Grid on the market.
+          </Typography>
+          <GetStartedButtons
```

Comment:
The `Get started` button is linking to the general (MUI Core) getting started page. I figure that, in the MUI X page, we should have it linking to Data Grid's version instead: `components/data-grid/getting-started/`